### PR TITLE
a new version for captchagen

### PR DESCRIPTION
**Pull request by eidonjoe**
1. Add drawImage and drawPoint function into project
2. Give a default value for every function
3. All configurable parameters when make a captch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,21 +26,31 @@ This relies on a native module called node-canvas. You will need to install cair
...
 ```javascript
-var captchagen = require('captchagen');
-
-// optional object arg with keys: height, width, text, font
-var captcha = captchagen.create();
-
-captcha.text();     // Returns captcha text (randomly generated by default)
-captcha.height();   // Returns captcha height (150 by default)
-captcha.width();    // Returns captcha width (300 by default)
+var captchagen = require('node-captchagen');
```
Comment (contra) [inline README.md]: node-captchagen?

File: README.md [hunk 2]
```diff
@@ -25,21 +26,31 @@ This relies on a native module called node-canvas. You will need to install cair
...
+var fs = require('fs');
```
Comment (contra) [inline README.md]: why have fs in the sample?

File: index.js [hunk 3]
```diff
@@ -21,12 +25,40 @@ module.exports = {
   generateText: generateText,
   
   // use default settings
-  create: function (opt) {
+  create: function (opt,drawArguments) {
```
Comment (contra) [inline index.js]: formatting

File: package.json [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
-   "name":"captchagen",
+   "name":"node-captchagen",
    "description":"Captcha generator",
-   "version":"1.1.0",
-   "homepage":"http://github.com/wearefractal/captchagen",
-   "repository":"git://github.com/wearefractal/captchagen.git",
-   "author":"Fractal <contact@wearefractal.com> (http://wearefractal.com/)",
+   "version":"1.2.1",
+   "homepage":"http://github.com/eidonjoe/captchagen",
+   "repository":"git://github.com/eidonjoe/captchagen.git",
```
Comment (contra) [inline package.json]: you changed the author? this is a license infringement

File: package.json [hunk 5]
```diff
@@ -1,10 +1,10 @@
 {
-   "name":"captchagen",
+   "name":"node-captchagen",
```
Comment (contra) [inline package.json]: changed module name?

Comment (eidonjoe) [inline package.json]: sorry for this ,this is my first pr in github,maybe make a lot of mistakes. I just want to use it Immediately,so I push a new module to npm. I know you are the owner.And if you republish a new module , I will continue use it and delete mine.i'm so sorry.

File: package.json (hunk 4 above)
Comment (eidonjoe) [inline package.json]: can i change back to you and create a new pr?

Comment (contra) [inline package.json]: @eidonjoe you can make changes to a PR by pushing changes to the PR branch

**Comment by contra** [comment]
Thanks for the PR! I left comments with some questions

**Pull request closed without merging**